internal/test/e2e/testRunner.go

```diff
@@ -134,6 +135,10 @@ func (v *VSphereTestRunner) setEnvironment() (map[string]string, error) {
 	}
 	envMap[govcInsecure] = strconv.FormatBool(v.Insecure)
 
+	if err := os.Setenv(govcDatacenterKey, v.Datacenter); err != nil {
```

Comment:
Actually, all you need is to set the value in the map.

`envMap[govcDatacenterKey] = v.Datacenter`

Reply:
Just FYI, this map gets passed directly to the govc container environment: https://github.com/aws/eks-anywhere/blob/5653615245e6bf719ff2057f23200ee8d4d57037/pkg/executables/govc.go#L490

I pass it directly to ensure it picks up the vpshere info for the lab rather than vmc. Otherwise if we are all setting/referencing the same default env vars there will be race conditions during the build since the test runners are created on same CodeBuild agent machine. This is easier to explain via voice so I'm open to chat if you have questions.

Reply:
Forgot about this sorry